### DISCUSSION TITLE
Edge canary 手机端支持扩展啦

### DISCUSSION THREAD

**Discussion by Curious-r** [immersive-translate/immersive-translate#1648] [General]
目前可以通过开启开发者选项然后直接上传crx文件的方式安装。我试了下，很好用，沉浸式翻译本身对移动端优化良好。
希望edge正式版推出这个功能之后，沉浸式翻译能进驻edge移动端扩展商店！

**Comment by Curious-r** [comment]
也可以用edge store的id安装，沉浸式翻译的id是amkbmndfnliijdhojkpoglbnaaahippg

**Comment by theowenyoung** [comment]
Edge 官方已经邀请了沉浸式翻译成为首批扩展的合作者，到时候应该可以在 Edge 中第一批使用。

**Reply by theowenyoung** [reply]
![222](https://github.com/immersive-translate/immersive-translate/assets/62473795/b2dc11c3-39a6-4761-af06-780cd87ac4ef)
